src/Exception/ApiException.php

```diff
@@ -50,7 +50,7 @@ public function __construct(GuzzleRequestException $previous, $message = '')
 
         if ($this->previous->hasResponse()) {
             $this->response = $this->previous->getResponse();
-            $this->requestId = $this->response->getHeader('X-Contentful-Request-Id')[0];
+            $this->requestId = $this->response->getHeaderLine('X-Contentful-Request-Id');
```

Comment:
Honestly I'd rather patch the coverage tool to let this header pass through (which would make a lot of sense anyway) than going down the path of fixing possible header issues (and reverting this change), what do you think?